app/scripts/modules/kubernetes/serverGroup/configure/configuration.service.js

```diff
@@ -17,7 +17,15 @@ module.exports = angular.module('spinnaker.serverGroup.configure.kubernetes.conf
 
       // this ensures we get the images we need when cloning or copying a server group template.
       let queries = command.containers
-        .map(c => grabImageAndTag(c.imageDescription.imageId));
+        .filter(c => {
```

Comment:
This filter handles all three cases correctly on our instance
deploy stage with trigger
deploy stage with find image stage
clone server group